apps/rag/app/app.py

```diff
@@ -307,6 +381,7 @@ async def main(args):
     logger.info("Starting RAG API")
     logger.info(f"Log level: {args.log_level}")
     logger.info(f"Input ID: {args.input}")
+    logger.info(args)
```

Comment:
This puts credentials into the log.

Reply:
OK will remove. Was just an aid to let me know what was being passed around